### PR TITLE
[wasm] Turn Wasm exception handling on by default and use WasmEnableExceptio…

### PR DESCRIPTION
**Pull request by lewing**
…nHandling as the default value for WasmEnableSIMD

### CODE REVIEW

File: src/mono/wasm/build/WasmApp.targets [hunk 1]
```diff
@@ -99,8 +99,9 @@
 
   <PropertyGroup>
     <WasmDedup Condition="'$(WasmDedup)' == ''">true</WasmDedup>
-    <WasmEnableExceptionHandling Condition="'$(WasmEnableExceptionHandling)' == ''">false</WasmEnableExceptionHandling>
-    <WasmEnableSIMD Condition="'$(WasmEnableSIMD)' == ''">true</WasmEnableSIMD>
+    <!-- Post Wasm MVP features -->
+    <WasmEnableExceptionHandling Condition="'$(WasmEnableExceptionHandling)' == ''">true</WasmEnableExceptionHandling>
+    <WasmEnableSIMD Condition="'$(WasmEnableSIMD)' == ''">$(WasmEnableExceptionHandling)</WasmEnableSIMD>
```
Comment (radekdoulik) [inline src/mono/wasm/build/WasmApp.targets:104]: curious: why to base the default SIMD value on EH value?

**Comment by kg** [comment]
Looks fine other than radek's comment

**Comment by lewing** [comment]
@radekdoulik should we change the benchmarks to explicitly set these settings based on their name/intension?

**Review by kg — APPROVED** [review]

File: src/mono/wasm/build/WasmApp.targets (hunk 1 above)
Comment (radekdoulik) [reply src/mono/wasm/build/WasmApp.targets:104]: > EH is more broadly compatible/deployed than SIMD

only in Safari. Chrome, Firefox and nodejs have SIMD support longer than EH. (https://webassembly.org/roadmap/)

I think the new behavior is a bit confusing. Maybe we could introduce a new property, like `WasmEnableAdvancedFeatures` or `WasmEnableNextGen` to switch the defaults of SIMD, EH and other non 1.1 features. What do you think?

**Pull request merged**